src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -278,7 +278,9 @@
   - name: Create a rule to allow a specific address-range.
     text: az storage account network-rule add -g myRg --account-name mystorageaccount --ip-address 23.45.1.0/24
   - name: Create a rule to allow access for a subnet.
-    text: az storage account network-rule add -g myRg --account-name mystorageaccount --vnet myvnet --subnet mysubnet
+    text: az storage account network-rule add -g myRg --account-name mystorageaccount --vnet-name myvnet --subnet mysubnet
+  - name: Create a rule to allow access for a subnet in another resource group.
+    text: az storage account network-rule add -g myRg --account-name mystorageaccount  --subnet $subnetId
```

Comment:
`$subnetId` It's not appropriate to include `$`, right?

Reply:
It is placeholder. resource id is too long to put it in example.

Reply:
OK